ajax/libs/cache.adderall/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "cache.adderall",
+  "version": "0.1.0",
+  "description": "An impatient version of cache.addAll()",
+  "homepage": "http://CacheAdderall.com",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/TalAter/cache.adderall.git"
+  },
+  "keywords": [
+    "CacheStorage",
+    "cache",
+    "storage",
+    "caching",
+    "service workers",
+    "serviceworker"
+  ],
+  "author": "Tal Ater <tal@talater.com> (https://www.talater.com/)",
+  "license": "MIT",
+  "autoupdate": {
```

Comment:
@TalAter We prefer to use npm auto-update config if the lib has been published on NPM. Please use npm auto-update config instead of git. Thanks. Ref: https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md